app/models/manageiq/providers/network_manager.rb

```diff
@@ -25,6 +25,21 @@ class << model_name
     has_many :load_balancer_health_checks,        :foreign_key => :ems_id, :dependent => :destroy
     has_many :load_balancer_health_check_members, :through => :load_balancer_health_checks
 
+    # Generates ORDER BY ((SELECT COUNT(*) FROM "vms" WHERE "ext_management_systems"."parent_ems_id" = "vms"."ems_id"))
```

Comment:
a. is it possible to define this higher up in the inheritance chain?

rewrite:

```ruby
# Uses "ext_management_systems"."parent_ems_id" instead of "ext_management_systems"."id"
#
# ORDER BY ((
#   SELECT COUNT(*)
#   FROM "vms"
#   WHERE "ext_management_systems"."parent_ems_id" = "vms"."ems_id"
# ))
```